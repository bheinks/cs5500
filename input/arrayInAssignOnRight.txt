(* Cannot have entire array on rhs of assignment *)

program p;

var A : array [1..5] of integer;
    x : integer;

begin
  x := A;
  write('!')
end.

