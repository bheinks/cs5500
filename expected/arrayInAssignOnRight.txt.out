Line 9: Array variable must be indexed
